mysql-test/suite/innodb_fts/r/plugin_debug.result

```diff
@@ -0,0 +1,306 @@
+4	Optimizing MySQL	In this tutorial we will show ...
+5	1001 MySQL Tricks	How to use full-text search engine
+SELECT * FROM articles WHERE
+MATCH(title, body) AGAINST('"for mysql"' IN BOOLEAN MODE);
+id	title	body
+1	MySQL Tutorial	DBMS stands for MySQL DataBase ...
+2	MySQL Tutorial	DBMS stands for MySQL good one DataBase ...
+3	How To Use MySQL Well	After you went through a ...
+4	Optimizing MySQL	In this tutorial we will show ...
+5	1001 MySQL Tricks	How to use full-text search engine
+SELECT * FROM articles WHERE
+MATCH(title, body) AGAINST('"mysql database"' IN BOOLEAN MODE);
+id	title	body
+SELECT * FROM articles WHERE
+MATCH(title, body) AGAINST('"full text search"' IN BOOLEAN MODE);
+id	title	body
+SELECT * FROM articles WHERE
+MATCH(title, body) AGAINST('"full text search engine"' IN BOOLEAN MODE);
+id	title	body
+SELECT * FROM articles WHERE
+MATCH(title, body) AGAINST('+"dbms stands" -good' IN BOOLEAN MODE);
+id	title	body
+1	MySQL Tutorial	DBMS stands for MySQL DataBase ...
+SELECT * FROM articles WHERE
+MATCH(title, body) AGAINST('+("dbms stands") -good' IN BOOLEAN MODE);
+id	title	body
+1	MySQL Tutorial	DBMS stands for MySQL DataBase ...
+SELECT * FROM articles WHERE
+MATCH(title, body) AGAINST('+("dbms stands" search) -good' IN BOOLEAN MODE);
+id	title	body
+5	1001 MySQL Tricks	How to use full-text search engine
+1	MySQL Tutorial	DBMS stands for MySQL DataBase ...
+SELECT * FROM articles WHERE
+MATCH(title, body) AGAINST('+("dbms stands" "full text") -good' IN BOOLEAN MODE);
+id	title	body
+1	MySQL Tutorial	DBMS stands for MySQL DataBase ...
+SELECT * FROM articles WHERE
+MATCH(title, body) AGAINST('("msyql database")@3' IN BOOLEAN MODE);
+id	title	body
+DROP TABLE articles;
+SET NAMES utf8;
+# Test Part 7: Test Different Charset
+CREATE TABLE articles (
+id INT UNSIGNED AUTO_INCREMENT NOT NULL PRIMARY KEY,
+title VARCHAR(200),
+body TEXT,
+FULLTEXT (title, body) WITH PARSER simple_parser
+) ENGINE=InnoDB DEFAULT CHARACTER SET gb2312 COLLATE gb2312_chinese_ci;
+INSERT INTO articles (title, body) VALUES
+('数据库 是 数据 的 结构化 集合','它 可以 是 任何 东西'),
+('从 简单 的 购物清单 到 画展','或 企业网络 中 的 海量信息'),
+('要 想 将 数据 添加 到 数据库','或 访问、处理 计算机 数据库 中 保存 >的 数据'),
+('需要 使用 数据库 管理系统','计算机 是 处理 大量 数据 的 理想 工具');
+SELECT * FROM articles WHERE
+MATCH(title, body) AGAINST('数据库');
+id	title	body
+3	要 想 将 数据 添加 到 数据库	或 访问、处理 计算机 数据库 中 保存 >的 数据
+1	数据库 是 数据 的 结构化 集合	它 可以 是 任何 东西
+4	需要 使用 数据库 管理系统	计算机 是 处理 大量 数据 的 理想 工具
+SELECT * FROM articles WHERE
+MATCH(title, body) AGAINST('数据');
+id	title	body
+SELECT * FROM articles WHERE
+MATCH(title, body) AGAINST('数据*');
+id	title	body
+3	要 想 将 数据 添加 到 数据库	或 访问、处理 计算机 数据库 中 保存 >的 数据
+1	数据库 是 数据 的 结构化 集合	它 可以 是 任何 东西
+4	需要 使用 数据库 管理系统	计算机 是 处理 大量 数据 的 理想 工具
+SELECT * FROM articles WHERE
+MATCH(title, body) AGAINST('+数据库 -计算机');
+id	title	body
+1	数据库 是 数据 的 结构化 集合	它 可以 是 任何 东西
+SELECT * FROM articles WHERE
+MATCH(title, body) AGAINST('"计算机 数据库"' IN BOOLEAN MODE);
+id	title	body
+DROP TABLE articles;
```

Comment:
In MySQL 5.7, this would return the row where we inadvertently injected the `>` character. But, as far as I can tell, the character was not added in the middle of this phrase.

The commit message does mention some missing results, but I think it would be clearer if it used some specific syntax to refer to that. I see that many but not all `IN BOOLEAN MODE` queries return an empty result or fewer results than MySQL. Is there a more precise way to explain this difference?